### ISSUE TITLE
Crop window too large for image when image dimensions match crop dimensions

### ISSUE THREAD

**Issue by butterybread** [AllanBishop/angular-img-cropper#42]
I found a bug where the crop window behaves very weird.
When the **crop-width** & **crop-height** dimension match the image resolution, the crop tool is out sort of out of bounds. But this only happens when the crop dimensions are larger than the canvas size. Also only happens if the canvas was empty before the image loaded.
#### Cases

(Here I have the width & height the same, but they could be different as long as they match the image dimensions. The canvas here is 500x300, but can be anything.)
- width&height: 200 | image: 200x200 => works fine.
- width&height: 800 | image: 800x800 => BUG
#### Buggy Behaviour
1. Initially no crop corners are shown.
   ![step1](https://cloud.githubusercontent.com/assets/2602506/9681944/4cbeda10-5302-11e5-914d-a6cbd950cebf.png)
2. When you click the cropper, 1 corner appears and the drag icon is gone.
   ![step2](https://cloud.githubusercontent.com/assets/2602506/9681957/63639300-5302-11e5-8747-5776ec6b3615.png)
3. The produced cropped image is also rather incorrect. The src image is scaled down and positioned in the upper left corner of the produced image.
   ![step3](https://cloud.githubusercontent.com/assets/2602506/9681934/34173b42-5302-11e5-8562-3ab7a1852739.png)
#### Demo

I changed the demo example to show the bug. This example uses crop-width=888 & crop-height=777. But you can always try with different dimensions.
http://output.jsbin.com/rilodalami/1
Easy dummy image generator: http://dummyimage.com/
image: http://dummyimage.com/888x777

Great work on the plugin. I really like it. Hope you can fix this bug. Thanks!

**Comment by albertonaperijr** [comment]
I encountered the same bug. Any fix to this issue? Thanks!

**Comment by mgoodfellow** [comment]
This has long since been fixed, I have been having the same problem too - I ended up digging through and checking all the historical commits - there have been loads of fixes since the v1.1.0 release!

https://github.com/AllanBishop/angular-img-cropper/commit/f6810766bae3a54bba92351a0943c50ca8134ff3

Commits since last release: https://github.com/AllanBishop/angular-img-cropper/compare/v1.1.0...master

Basically, download the master and use that
